File: configuration/src/test/java/com/synopsys/integration/configuration/property/types/path/SimplePathResolverTest.java
package com.synopsys.integration.configuration.property.types.path;

import java.nio.file.Paths;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class SimplePathResolverTest {
    @Test
    public void resolvePathTest() {
        PathResolver simplePathResolver = new SimplePathResolver();
        Assertions.assertEquals(Paths.get("/simple/test").toAbsolutePath(), simplePathResolver.resolvePath("/simple/test"));
    }
}

File: configuration/src/test/java/com/synopsys/integration/configuration/property/types/path/SimplePathResolverTest.kt
package com.synopsys.integration.configuration.property.types.path

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import java.nio.file.Paths

class SimplePathResolverTest {
    @Test
    fun resolvePathTest() {
        val simplePathResolver = SimplePathResolver()
        Assertions.assertEquals(Paths.get("/simple/test"), simplePathResolver.resolvePath("/simple/test"))
    }
}